### PR TITLE
Update: color and line-height removed

### PR DESCRIPTION
**Pull request by p-musi**
Ticket: https://jira.tapestry.support/browse/TORO-54724 <br>

Extra line-height and color classes are removed

### CODE REVIEW

File: src/global/color/index.json [hunk 1]
```diff
@@ -79,97 +79,19 @@
...
-        "value": "rgba(0255,255,255,0.6)"
-      },
-      "70": {
-        "value": "rgba(0255,255,255,0.7)"
-      },
-      "80": {
-        "value": "rgba(255,255,255,0.8)"
-      },
-      "90": {
-        "value": "rgba(255,255,255,0.9)"
```
Comment (p-musi) [inline src/global/color/index.json:108]: Hi @ejaz-tapestry,

We found that SFCC is mapping these values, so we don't want to remove them anymore.  Can we undo the changes for all of these?

**Review by aoverstreet-tapestry — APPROVED** [review]
